lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -107,18 +125,39 @@ def main(argv=None):
     args = parser.parse_args(args=argv)
 
     # Load Cubes
-    historic_forecast = load_cube(args.historic_filepath)
-    truth = load_cube(args.truth_filepath)
+    historic_forecast = load_cube_or_none(args.historic_filepath)
```

Comment:
FYI this function may be disappearing in @neilCrosswaite's work.

Reply:
It is disappearing as of #941 which may well be merged before Gavin returns. So yes, load_cube_or_none, if you want the return None behaviour is now:
```
load_cube(filepath, allow_none=True)
```

Reply:
Yes, I've now updated this to use the `load_cube` function with the `allow_none` keyword argument.